module/module.go

```diff
@@ -417,10 +417,10 @@ func (m *Module) Ready(ctx context.Context, req *pb.ReadyRequest) (*pb.ReadyResp
 
 // AddResource receives the component/service configuration from the parent.
 func (m *Module) AddResource(ctx context.Context, req *pb.AddResourceRequest) (*pb.AddResourceResponse, error) {
-	select {
-	case <-m.pcReady:
-	case <-m.pcFailed:
-	}
+	// select {
```

Comment:
1. Should these comments be removed?
2. If the module is no longer going to wait for pcReady should we remove them from the module struct?

Reply:
Nice callout. This was an accident. I thought I was in `modmanager/manager.go` when I searched for these channels to move them. Then I realized I was in the wrong file and forgot to undo this part.

I've added them back in as-is. The intention was to not touch the module.go stuff in this PR unless necessary. I suspect this file may want cleanup too. But not in the same PR as modmanager.